librz/core/gdiff.c

```diff
@@ -9,7 +9,7 @@
 #include <rz_util.h>
 #include <rz_core.h>
 
-RZ_API int rz_core_gdiff_fcn(RzCore *c, ut64 addr, ut64 addr2) {
+RZ_API bool rz_core_gdiff_fcn(RzCore *c, ut64 addr, ut64 addr2) {
```

Comment:
I think it would be nice to explain the difference between `_fcn()` and `_fcn2()`